### ISSUE TITLE
Question: Is it normal to see WARN: mempool sync failed: failed to get mempool txids in sequence?

### ISSUE THREAD

**Issue by tadeubas** [romanz/electrs#738]
Is it normal to see this `WARN: mempool sync failed: failed to get mempool txids` is sequence sometimes? It always happen in sequence and then stops warning after  ~1minute... Do this means my bitcoind should have more `rpcthreads` or `rpcworkqueue` or increased the time on `rpcservertimeout`?

![image](https://user-images.githubusercontent.com/5518496/175925868-b9f22716-9301-4ccd-ab4f-7e5f4f21c996.png)

**Comment by smartm0use** [comment]
The same happen to me, any idea?

**Comment by tadeubas** [comment]
Don't know, but it is just a warning, so maybe this is expected to happen...

**Comment by romanz** [comment]
electrs uses the following Bitcoin Core RPCs to sync the mempool: 
 - https://docs.rs/bitcoincore-rpc/0.16.0/bitcoincore_rpc/trait.RpcApi.html#method.get_raw_mempool
 - https://docs.rs/bitcoincore-rpc/0.16.0/bitcoincore_rpc/trait.RpcApi.html#method.get_raw_transaction
 - https://docs.rs/bitcoincore-rpc/0.16.0/bitcoincore_rpc/trait.RpcApi.html#method.get_mempool_entry

Since their invocations are not synchronized, it's possible that a transaction will RBF-ed or confirmed, causing the next RPCs to "miss" it - resulting in the above message.

TL;DR: it's not a real problem, just an artifact of the existing mempool sync implementation
